lib/modules/manager/cdnurl/readme.md

```diff
@@ -11,6 +11,6 @@ For example:
 
 <!-- prettier-ignore -->
 !!! warning
-    This manager isn't aware of subresource integrity (SRI) hashes.
-    It searches for and replaces _any_ matching URL it finds, without considering things like script integrity hashes.
-    Use the `html` manager if you need SRI updating.
+    This manager does _not_ read or update Subresource Integrity (SRI) hashes.
```

Comment:
_Why_ does the `cdnurl` manager _not_ read or update SRI hashes? I think it's a good idea to use SRI hashes, so you know the CDN delivers the correct dependency? 

Is there a design/code issue that prevents the `cdnurl` manager from working with SRI hashes?

Reply:
I forget. It's _probably_ for a good reason, and nobody's ever asked for it in all these years, so I would prefer to keep it as is.